lib/redcord/prepared_redis.rb

```diff
@@ -12,11 +12,16 @@ class Redcord::PreparedRedis < Redis
     ).returns(Integer)
   end
   def create_hash_returning_id(key, args)
-    evalsha(
-      self.class.server_script_shas[:create_hash_returning_id],
-      keys: [key],
-      argv: args.to_a.flatten,
-    ).to_i
+    Redcord::Base.trace(
+      'create_hash_returning_id',
+      resource: key,
```

Comment:
nit: I'd consider having different hash arguments for different types, eg `key:` and `model:`, it might be easier for a subscriber to deal with that way?

Reply:
good idea!